app/models/department.rb

```diff
@@ -0,0 +1,6 @@
+class Department < ApplicationRecord
+  validates :dep_name, presence: true
+  validates :dep_name, uniqueness: true
```

Comment:
If table name is `departments` then I think, there is no need to add `dep` in `name` column. Thoughts?
```suggestion
  validates :name, presence: true
  validates :name, uniqueness: true
```